src/server/src/test/java/com/spotify/reaper/resources/view/NodesStatusTest.java

```diff
@@ -37,18 +37,17 @@ public void testParseEndpointStatusString(){
         .append("/127.0.0.2")
         .append("  generation:1497347537 ")
         .append("  heartbeat:27077 ")
-        .append("  STATUS:14:NORMAL,-3074457345618258603 ")
-        .append("  LOAD:26921:3988763.0 ")
-        .append("  SCHEMA:10:2de0af6a-bf86-38e0-b62b-474ff6aefb51 ")
-        .append("  DC:6:datacenter2 ")
-        .append("  RACK:8:rack2 ")
-        .append("  RELEASE_VERSION:4:3.0.8 ")
-        .append("  RPC_ADDRESS:3:127.0.0.2 ")
-        .append("  SEVERITY:27076:0.0 ")
-        .append("  NET_VERSION:1:10 ")
-        .append("  HOST_ID:2:08f819b5-d96f-444e-9d4d-ec4136e1b716 ")
-        .append("  RPC_READY:44:true ")
-        .append("  TOKENS:13:<hidden> \r")
+        .append("  STATUS:NORMAL,-3074457345618258603 ")
+        .append("  LOAD:3988763.0 ")
+        .append("  SCHEMA:2de0af6a-bf86-38e0-b62b-474ff6aefb51 ")
+        .append("  DC:datacenter2 ")
+        .append("  RACK:rack2 ")
+        .append("  RELEASE_VERSION:3.0.8 ")
+        .append("  RPC_ADDRESS:127.0.0.2 ")
+        .append("  SEVERITY:0.0 ")
+        .append("  NET_VERSION:10 ")
+        .append("  HOST_ID:08f819b5-d96f-444e-9d4d-ec4136e1b716 ")
+        .append("  RPC_READY:true \r")
```

Comment:
nit-pick: i would have duplicated the test method, so there was one for testing 2.1 gossipinfo and one for 2.2 gossipinfo. that would make intention and test failures clearer.